atom/browser/common_web_contents_delegate.cc

```diff
@@ -180,12 +180,13 @@ void CommonWebContentsDelegate::SetOwnerWindow(NativeWindow* owner_window) {
 void CommonWebContentsDelegate::SetOwnerWindow(
     content::WebContents* web_contents, NativeWindow* owner_window) {
   owner_window_ = owner_window ? owner_window->GetWeakPtr() : nullptr;
-  NativeWindowRelay* relay = new NativeWindowRelay(owner_window_);
+  auto relay = base::MakeUnique<NativeWindowRelay>(owner_window_);
+  auto relayKey = relay->key;
```

Comment:
According to the Style Guide the name should be `relay_key`.
https://google.github.io/styleguide/cppguide.html#Variable_Names